static/js/server_events_dispatch.js

```diff
@@ -429,15 +429,18 @@ export function dispatch_normal_event(event) {
             switch (event.op) {
                 case "add":
                     people.add_active_user(event.person);
+                    settings_account.maybe_update_deactivate_account_button();
                     break;
                 case "remove":
                     people.deactivate(event.person);
                     stream_events.remove_deactivated_user_from_all_streams(event.person.user_id);
                     settings_users.update_view_on_deactivate(event.person.user_id);
                     buddy_list.maybe_remove_key({key: event.person.user_id});
+                    settings_account.maybe_update_deactivate_account_button();
                     break;
                 case "update":
                     user_events.update_person(event.person);
+                    settings_account.maybe_update_deactivate_account_button();
```

Comment:
We want to check for this update for all three cases (add, remove and update), but necessarily for the error case.

Reply:
There are a couple of places where we call the common function for all the cases whereas for one case we call it once (and thus is called in the error case as well). Not sure what we prefer, but I find this, i.e. calling the function for all cases,  more clearer.